### PR TITLE
G_CM_60 | CM Acknowledging the data request

### PR DESCRIPTION
**Pull request by Saisree9**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java  1
         

Clones added
============
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java  1
- src/main/java/in/projecteka/consentmanager/dataflow/DataFlowRequester.java  2
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterUserJourneyTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5717116&bid=18708332)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java  1
         

Clones added
============
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java  1
- src/main/java/in/projecteka/consentmanager/dataflow/DataFlowRequester.java  2
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterUserJourneyTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5717116&bid=18708332)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java  1
         

Clones added
============
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java  1
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterUserJourneyTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5717116&bid=18708332)

File: src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+
+import static in.projecteka.consentmanager.common.Constants.HDR_HIU_ID;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+@AllArgsConstructor
+public class DataFlowRequestClient {
+    private static final String DATA_FLOW_REQUEST_URL_PATH = "/health-information/cm/on-request";
```
Comment (gayatrijagtap) [inline src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java:19]: can't we take this url from constants?

Comment (MounicaSagi) [reply src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java]: done.

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/main/java/in/projecteka/consentmanager/clients/DataFlowRequestClient.java  1
         

Clones added
============
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterUserJourneyTest.java  1
- src/test/java/in/projecteka/consentmanager/dataflow/DataFlowRequesterTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/consent-manager/pullRequest?prid=5717116&bid=18708332)

**Pull request merged**